sdcm/cluster_k8s/__init__.py

```diff
@@ -839,6 +864,26 @@ def deploy_scylla_cluster(self, node_pool: CloudK8sNodePool = None, node_prepare
         self.init_scylla_config_map()
         self.scylla_restart_required = False
 
+        # Deploy NodeConfig resource that will prepare K8S nodes for Scylla pods
```

Comment:
Please convert this comment into log message

Reply:
ok, will do

Reply:
done